microsoft-edge/extensions-chromium/publish/api/using-addons-publish-api.md

```diff
@@ -0,0 +1,220 @@
+---
+description: REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons website.
+title: Using the Microsoft Edge Add-ons Publish API
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/10/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, publish api, add-ons publish api
+---
+# Using the Microsoft Edge Add-ons Publish API
+
+<!-- find and resolve: tbd, html comments, https://addons.edge.microsoft.com/api -->
+
+This article, along with the [Add-ons Publish API Reference][AddonsPublishAPIRef], provides an overview of the proposed Microsoft Edge Add-ons Publish API.  We look forward to your suggestions and feedback on the proposed API contracts.  Please submit your feedback at (_link TBD_).
```

Comment:
Where can people now provide feedback?

Reply:
@mikehoffms we will publish a tech community blog on this as soon as the documentation is live. That might be the best place, though it requires sign in with an MSA account. 

Is there  away to get comments on GitHub itself?

Reply:
Yes, eg go to https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/ then click Edit to go to the docs repo at github, then click Issues, you end up at https://github.com/MicrosoftDocs/edge-developer/issues

Reply:
I added a link to Issues, like other such links in this repo.